thoth/common/openshift.py

```diff
@@ -1709,7 +1709,7 @@ def get_mi_repositories_and_organizations(
             if repositories is not None and repositories != ""
             else []
         )
-        return (repos, orgs)
+        return ([r.split() for r in repos], [o.split() for o in orgs])
```

Comment:
is the output seperate with commas "," ? sorry i don't remember how the repos were structured.
this would split with spaces , is that what we want ?

Reply:
yes exactly

Reply:
names are divided by commas, however there can be optional number of spaces in between the name and comma